raiden/tests/integration/test_provisioning.py

```diff
@@ -5,10 +5,75 @@
 
 from raiden.api.python import RaidenAPI
 from raiden.tests.utils.blockchain import wait_until_block
+from raiden.transfer.state import (
+    CHANNEL_STATE_SETTLED,
+)
 
 log = slogging.getLogger(__name__)
 
 
+@pytest.mark.parametrize('number_of_nodes', [2])
+@pytest.mark.parametrize('number_of_tokens', [1])
+@pytest.mark.parametrize('channels_per_node', [1])
+@pytest.mark.parametrize('settle_timeout', [6])
+@pytest.mark.parametrize('reveal_timeout', [3])
+def test_close_raiden_app_gracefully(
+    raiden_network,
+    token_addresses,
+    settle_timeout,
+    blockchain_type
+):
+    if blockchain_type == 'tester':
```

Comment:
why? Because you would need different logic to wait until the settlement blocks have been "mined"?